sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -292,6 +293,19 @@ public ConfigurationClientBuilder retryPolicy(RetryPolicy retryPolicy) {
         return this;
     }
 
+    /**
+     * Sets the {@link ServiceVersion} that is used when making API requests.
+     *
+     * The latest service version will be used if not provided {@link ServiceVersion}
+     *
+     * @param version {@link ServiceVersion} of the service to be used when making requests.
+     * @return The updated ConfigurationClientBuilder object.
+     */
+    public ConfigurationClientBuilder serviceVersion(ServiceVersion version) {
```

Comment:
The `buildClient` and `buildAsyncClient` methods should pass this version to the client constructor.

The client must use the appropriate `ServiceInterface` class to pass to `RestProxy.create();`. This can be done as next step but the version should be passed on to the client.

Reply:
Added Added `serviceVersion` field to async client constructors.